web/src/inbox_ui.ts

```diff
@@ -1626,7 +1785,42 @@ export function change_focused_element(input_key: string): boolean {
     return false;
 }
 
+function bulk_insert_channel_folders(channel_folders: Set<number>): void {
+    sort_channel_folders();
+    // Insert missing channel folders.
+    let index = 0;
+    let previous_folder_id;
+    for (const [folder_id, channel_folder] of channel_folders_dict) {
+        if (channel_folders.has(folder_id)) {
+            const $channel_folder = render_inbox_folder_with_channels({
```

Comment:
`$folder_row_html` seems like a better name -- we generally want `_html` suffixes on rendered HTML. And again `folder_context` for the loop variable.